README.md

```diff
@@ -35,4 +35,52 @@ The Triton backend for the
 can learn more about Triton backends in the [backend
 repo](https://github.com/triton-inference-server/backend).  Ask
 questions or report problems in the main Triton [issues
-page](https://github.com/triton-inference-server/server/issues).
+page](https://github.com/triton-inference-server/server/issues). The backend
+is designed to run models in Intermediate Representation (IR). See [here](https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html) for instruction to convert a model to IR format. The backend is implemented using openVINO C++ API. Auto completion of the model config is not supported in the backend and complete `config.pbtxt` must be provided with the model.
+
+
+Where can I ask general questions about Triton and Triton backends?
+Be sure to read all the information below as well as the [general
+Triton documentation](https://github.com/triton-inference-server/server#triton-inference-server)
+available in the main [server](https://github.com/triton-inference-server/server)
+repo. If you don't find your answer there you can ask questions on the
+main Triton [issues page](https://github.com/triton-inference-server/server/issues).
+
+**Note:** OpenVINO backend is in early stage of developement and is not yet ready for general use.
+
+## Build the OpenVINO Backend
+
+Cmake 3.17 or higher is required. First install the required dependencies.
+
+```
+$ apt-get install patchelf rapidjson-dev python3-dev
+```
+
+Follow the steps below to build the backend shared library.
+
+```
+$ mkdir build
+$ cd build
+$ cmake -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install -DTRITON_BUILD_OPENVINO_VERSION=2021.2.200 -DTRITON_BUILD_CONTAINER_VERSION=20.12 ..
+$ make install
+```
+
+The following required Triton repositories will be pulled and used in
+the build. By default the "main" branch/tag will be used for each repo
+but the listed CMake argument can be used to override.
+
+* triton-inference-server/backend: -DTRITON_BACKEND_REPO_TAG=[tag]
+* triton-inference-server/core: -DTRITON_CORE_REPO_TAG=[tag]
+* triton-inference-server/common: -DTRITON_COMMON_REPO_TAG=[tag]
+
+## Advanced
+
+### Parameters
+
+The configuration is done through the `config.pbtxt` file. The parameters and their description are as follows.
+
```

Comment:
After describing the parameters show an example config.pbtxt parameters section setting some values.